lib/processors/bundlers/moduleBundler.js

```diff
@@ -37,6 +37,57 @@ class LocatorResourcePool extends ResourcePool {
 	}
 }
 
+/**
+ * A ModuleBundleDefinitionSection specifies the embedding technology ('provided', 'raw', 'preload' or 'require')
+ * and lists the resources that should be in- or excluded from the section.
+ * <p>
+ * <b>Module bundle section modes</b><br>
+ * <ul>
+ * 	<li>
+ * 		<code>provided</code>: A section of type 'provided' defines a set of modules that should not be included in the bundle file itself, but
+			which should be assumed to be already loaded (or 'provided') by the environment into which the bundle module is loaded.
+ * 	</li>
+ *	<li>
+		 <code>raw</code>: A raw section determines the set of modules that should be embedded,
+			sorts them according to their dependencies and writes them out 1:1 without any transformation or wrapping (raw). Only JavaScript sources
+				can be embedded in a raw section.
+	</li>
+ *	<li>
+		 <code>preload</code>: A preload section packages resources that should be stored in the preload cache in the client.
+			They can embed any textual resource type (JavaScript, CSS and Theme-Parameter files) that the bundling supports.
```

Comment:
> "JavaScript, CSS and Theme-Parameter files"

While this is true for the bundling, I don't think we are able to read CSS or Theme Parameter files from the preload, right?

A better list IMO would be "JavaScript, XML, JSON, properties"

Reply:
Done